doc/manuals/installation_and_administration_manual/streaming/stream_processing_sinfonier_api.md

```diff
@@ -0,0 +1,135 @@
+#<a name="top"></a>Stream processing: Sinfonier API
+
+Content:<br>
+
+* [Sinfonier API](#section1)
+* [Requisites](#section2)
+* [Install](#section3)
+* [Configure](#section4)
+* [References](#section5)
+
+##<a name="section1"></a>Sinfonier API
+
+Sinfonier API was develop to manage [Sinfonier Backend](https://github.com/sinfonier-project/sinfonier-backend) and deal with Apache Storm cluster. This software is part of [Sinfonier-Project](http://sinfonier-project.net)
+
+Main features are:
+
+* Compile modules
+* Manage Sinfonier Backend classpath
+* Start/Stop Storm toplogies
+* Storm Log management
+
+[Top](#top)
+
+##<a name="section2"></a>Requisites
+
+**Dependencies**
+
+Jinga2
+
+```sh
+$ cd /tmp/
+$ wget https://pypi.python.org/packages/source/J/Jinja2/Jinja2-2.8.tar.gz
+$ tar xvf Jinja2-2.8.tar.gz
+$ cd Jinja2-2.8/
+$ sudo python setup.py install
+```
+
+Crypto
+
+```sh
+$ sudo yum install python-crypto
+```
+
+PIP Dependencies
+
+```sh
+$ sudo pip install tornado requests pymongo simplegist paramiko scp BeautifulSoup4
+```
+
+[Top](#top)
+
+##<a name="section3"></a>Install
+
+```sh
+$ cd /opt
+$ git clone https://github.com/sinfonier-project/sinfonier-api.git
+$ python sinfonierapicore.py
+
+```
+
+[Top](#top)
+
+##<a name="section4"></a>Configure 
+
+**MONGO CONFIG**
+
+```python
```

Comment:
Are these Python senteces run on a Python shell? Or must they be put into a file (Iin that case, which one?)?